README.md

```diff
@@ -69,8 +67,4 @@ Go back to the plugin config section in your Magento instance, and enter the web
 
 ## Contact Us
 
-If you have any questions or concerns you can contact our support team at  support@degica.com.
-
-## Development
```

Comment:
Since we're creating the userguide from the README I removed the Development section, since that wouldn't make much sense to someone browsing the Magento store. We've still got the `dev_setup.md` file in the docs folder, but maybe there's a better way to surface the dev setup info to someone?

Reply:
I think its fine for now.